docs/java/how-to-develop-an-activity-definition-in-java.md

```diff
@@ -0,0 +1,110 @@
+---
+id: how-to-develop-an-activity-definition-in-java
+title: How to develop an Activity Definition in Java
+sidebar_label: Activity Definition
+description: In the Temporal Java SDK programming model, Activities are classes which implement the ActivityInterface Interface.
+tags:
+  - java
+  - developer-guide
+---
+
+In the Temporal Java SDK programming model, [Activities](/docs/concepts/what-is-an-activity) are classes which implement the Activity interface.
```

Comment:
```suggestion
In the Temporal Java SDK programming model, an [Activity Definition](/docs/concepts/what-is-an-activity) is the complete combination of a Java Class implementing a specially annotated Java Interface.
```

Curious if this resonates with you?

Reply:
Updated to "An [Activity Definition](/docs/concepts/what-is-an-activity) is a combination of the Temporal Java SDK [Activity](https://www.javadoc.io/static/io.temporal/temporal-sdk/0.19.0/io/temporal/activity/Activity.html) Class implementing a specially annotated interface."

May rephrase this later.